### DISCUSSION TITLE
Minor - Concept 11

### DISCUSSION THREAD

**Discussion by abulyomon** [OpenMined/courses#218] [Feedback] [answered]
Not sure how significant this may be: Concept 11 Video at t 2'40" @trask refers to information flow output while the context of discussion is input verification.

**Comment by ZSoumia** [comment, marked as answer]
Hi @abulyomon  the first minutes of that video were some sort of an introduction to the topic. A way to initiate the context of the discussion and link it to the past concepts visited before. So this is not an error.

**Reply by hwrdtm** [reply]
@abulyomon please let us know if we've answered all we can regarding this discussion / question!
